spring-data-jdbc-plus-repository/src/main/java/com/navercorp/spring/data/jdbc/plus/repository/support/JdbcPlusRepository.java

```diff
@@ -78,4 +84,28 @@ public <S extends T> S update(S instance) {
 	public <S extends T> List<S> updateAll(Iterable<S> entities) {
 		return entityOperations.updateAll(entities);
 	}
+
+	@Transactional
```

Comment:
Do we need this?

Reply:
@IAM20 

This annotation was unnecessary, so I removed it in commit 61ef550db16fca2c9350962e44ce959a267f7255.
Thanks!